entity/migrations/0012_entitygroupmembership_sort_order.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.5 on 2023-09-21 16:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('entity', '0011_entitygroup_logic_string'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='entitygroupmembership',
```

Comment:
Made this a separate migration because it's a separate model but I can combine it with the other if you'd rather group them by changeset